workflow/r/slim/v1/amazon.redshiftodbc.ini

```diff
@@ -0,0 +1,9 @@
+[Driver]
+## - Note that this default DriverManagerEncoding of UTF-32 is for iODBC.
+DriverManagerEncoding=UTF-16
+ErrorMessagesPath=/opt/amazon/redshiftodbc/ErrorMessages
+LogPath=[LogPath]
+SwapFilePath=/tmp
+
+#   iODBC
+ODBCInstLib=/usr/lib/x86_64-linux-gnu/libodbcinst.so
```

Comment:
Hey @saurabh0402 - can we confirm once if we have iOdbc installed/available. I remember for snowflake we are using unixODBC which we install explicitly. Also worth considering that we can use same for both snowflake and redshift.

Reply:
We can use both `unixODBC` and `iOdbc` for Redshift. Have updated the env as per `unixODBC`.